<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="Animaux">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>841</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>accueil</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>531</width>
        <height>351</height>
       </rect>
      </property>
      <property name="title">
       <string>Animal</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
       </property>
       <property name="text">
        <string> NOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
       </property>
       <property name="text">
        <string> RACE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}

</string>
       </property>
       <property name="text">
        <string> AGE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_age">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>190</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_emplacement">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>290</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="la_race">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>140</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>91</width>
         <height>20</height>
        </rect>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
       </property>
       <property name="text">
        <string>ID ANIMAL</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;xx&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
       </property>
       <property name="text">
        <string> DATE ENTREE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>290</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
       </property>
       <property name="text">
        <string> EMPLACEMENT</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_id">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>50</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_nom">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_age_5">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>113</width>
         <height>121</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="insererphoto">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>160</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
       </property>
       <property name="text">
        <string>inserer photo</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="la_date">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>240</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="Lienderenseignement">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>210</y>
        <width>151</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>Lien de renseignement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_supprimer">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="modifier">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_ajouter">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ImportPDF">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>120</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}
</string>
      </property>
      <property name="text">
       <string>Import PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="trier">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>0</y>
        <width>961</width>
        <height>501</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../background anim.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>groupBox</zorder>
     <zorder>Lienderenseignement</zorder>
     <zorder>pb_supprimer</zorder>
     <zorder>modifier</zorder>
     <zorder>pb_ajouter</zorder>
     <zorder>ImportPDF</zorder>
     <zorder>trier</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Mailing</string>
     </attribute>
     <widget class="QPushButton" name="mailing">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>370</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>mailing</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Destinaire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Objet</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_age_6">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_age_7">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>261</width>
        <height>21</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_age_8">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>210</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>-55</x>
        <y>-5</y>
        <width>981</width>
        <height>531</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../background anim.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_11</zorder>
     <zorder>mailing</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>lineEdit_age_6</zorder>
     <zorder>lineEdit_age_7</zorder>
     <zorder>lineEdit_age_8</zorder>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>afficher</string>
     </attribute>
     <widget class="QTableView" name="tab_animal">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>771</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-95</y>
        <width>1011</width>
        <height>961</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../background anim.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_12</zorder>
     <zorder>tab_animal</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1221</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
